openmp/runtime/src/include/omp-tools.h.var

```diff
@@ -60,6 +60,23 @@
     macro(ompt_get_target_info)             \
     macro(ompt_get_num_devices)
 
+#define FOREACH_OMPT_DEVICE_INQUIRY_FN(macro) \
```

Comment:
Downstream examples (e.g. here I like this location way better):
https://github.com/ROCm/llvm-project/blob/amd-staging/offload/include/OmptCommonDefs.h#L26-L46